File: Python/Problem001.py
"""
Project Euler, problem 1
Find the sum of all the multiples of 3 or 5 below 1000.

answer = 233168  # correct
"""

answer = 0

for x in range(0, 1000):
    if x % 3 == 0 or x % 5 == 0:
        answer += x

print(answer)

File: Java/src/Problem001.java
/* Project Euler, problem 1.
 * Find the sum of all the multiples of 3 or 5 below 1000.
 * 
 * answer = 233168;  //correct
 */

public class Problem001 {
	public static void main(String[] args){
		
		int answer = 0;
		
		for (int i = 1; i < 1000; i++) {
			if(i % 3 == 0 || i % 5 == 0){
				answer += i;
			}
		}
		System.out.println(answer);
	}
}
